### DISCUSSION TITLE
“AlDente.app” will damage your computer. You should move it to the Bin.

### DISCUSSION THREAD

**Discussion by CoachM** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#177] [General]
Just installed AlDante with brew this morning and I got the message in the title and can't start the app. Removed it with brew and downloaded the zip from here. Same problem.

What causes this? What can I do to use the app?

**Comment by cassus** [comment]
Related: https://github.com/davidwernhart/AlDente/issues/173

**Reply by manishchauhan3** [reply]
Same here. Unable to load the application. macOS 11.2.3
![Untitled](https://user-images.githubusercontent.com/80399489/110631348-6e5f8f80-81cc-11eb-8685-620595fecfc7.png)

**Comment by Samuels1950** [comment]
Same here. Was working earlier this morning (Rome Italy time) without issue. Came back after lunch and now experiencing this issue.

**Comment by Brewforce** [comment]
Same on an MBA M1, on macOS 11.2.3. It was working fine since yesterday, when I made my first install

**Comment by danfoerster** [comment]
Guess I did not reboot my Mac since updating macOS to 11.2.3 a couple days ago. After rebooting, AlDente no longer works for me either and gives me the same warning. Weird that it did work up to this point because the Computer restarts after updating as well. Odd.

**Comment by MatthiasKerbl** [comment]
Hi Guys,
We are very sorry for the inconvenience with AlDente.
A new fixed version was just released:

"Because of an expired signing certificate, macOS shows a security warning for older versions of AlDente.
This has been fixed with this new (notarized!) update.
Also, this version has been compiled to work with macOS versions Mojave and newer.

For anyone who still has problems (regarding old helper versions that are now marked as a "threat" by MacOS), please try the following steps:

- Delete /Library/LaunchDaemons/com.davidwernhart.Helper.plist

- Delete /Library/PrivilegedHelperTools/com.davidwernhart.Helper

- Restart your MacBook

- Install this new version of AlDente as usual"

Best regards,
David & Matthias
